package.json

```diff
@@ -2,12 +2,14 @@
   "license": "MIT",
   "scripts": {
     "clean": "npm prune && del-cli ./packages/*/dist ./packages/*/tsconfig.tsbuildinfo",
-    "compile": "tsc --build tsconfig.build.json",
-    "compile:clean": "tsc --build tsconfig.build.json --clean",
-    "compile:tests": "tsc --build tsconfig.test.json && tsc --build tsconfig.test.json --clean",
+    "build": "npm run build:types:clean && npm run check:types:tests && npm run build:types && npm run build:js",
```

Comment:
For some reason building the type declarations *after* building JS fails, needs to build type declarations first.